### ISSUE TITLE
Add cache for plex images

### ISSUE THREAD

**Issue by gugahoi** [gugahoi/maraschino#20]

**Comment by Wraithers** [comment]
Also, add blank thumbnails for media that doesn't have a thumbnail generated in Plex.

Just in case it helps, following is how each show up. Though it would probably be better to target it via class assigning if it has or a thumb or not.
- _With thumbnail:_
  
  `<img src="http://192.168.1.160:32400/library/metadata/56/thumb/1388595068" class="movie">`
- _Without thumbnail:_
  
  `<img src="http://192.168.1.160:32400" class="movie">`

**Comment by Wraithers** [comment]
Hey, I'd like to suggest the attached image to replace the poster.png for media without thumbnails, for consistency with Plex =)

Cropped & resized to match the file you used:
![poster](https://f.cloud.github.com/assets/4705057/2486032/31b7f6e0-b121-11e3-95cc-6182b0fc44b2.png)

(Sorry this is partially off-topic)

**Comment by gugahoi** [comment]
The thumb I used is already in Maraschino so more likely to be in cache. I dont know how it's looking in the module tho since I cant remove a thumb from my media and the random one that I populated in my library wont show up on my On Deck collection. Does it look bad?

future reference: plex has a blob REST service, might be better that using local cache:
blob:http%3A//plex.tv/594fe83f-33c5-45f9-a450-867efac651ac

**Comment by Wraithers** [comment]
Yup, it doesn't sit well with the rest of the Plex module, if already using similar icons and all, would be best to switch the poster over as well.

**Comment by gugahoi** [comment]
I have just added a proxy url in maraschino for plex images so now as long as maraschino can access the server it will be able to retrieve images anywhere in the world.
